config/defaults.conf

```diff
@@ -123,6 +123,7 @@ val_data_limit = 5000  // Maximum number of examples to be used during mid-train
                        // end of main.py that is invoked by do_full_eval.
 val_interval = 1000  // Number of steps between validation checks.
                      // Must be divisible by bpp_base (which is usually 1).
+max_epochs_per_task = None // If positive, maximum number of epochs (full pass over all training data) to perform
```

Comment:
Explain how this relates to max_vals. We run until _either_ criterion is hit, right? Might also be worth adding an option to set max_vals to None, with the same effect.

Reply:
Also, why 'per task'?